src/api/utils.ts

```diff
@@ -46,3 +47,25 @@ export const getTotal = (
   }).format(sum)
   return addSpaceBeforeUSDSymbol(formattedSum)
 }
+
+export const nFormatter = (num: number, digits = 2) => {
```

Comment:
Might be worth seeing if there is a community supported library for formatting numbers that suits our needs because it will likely be more robust

Reply:
i think this function works nice for current project needs. 
if we would need something more complicated, we can use http://numeraljs.com/